lib/config.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+
+config['stats'].storage.push({name: 'redis'});
+// generate stats reports at 1 minute intervals
+config['stats'].report.interval = 60 * 1000;
```

Comment:
This is the same value set in bedrock-stats.  Is it here just to make sure 60s is used?